pkgs/tools/security/osv-scanner/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  testers,
+  osv-scanner,
+}:
+buildGoModule rec {
+  pname = "osv-scanner";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-AftJS8IdKc1oDJfDTB/HMLHu7EAVnPGPR61SOlTnC2E=";
+  };
+
+  vendorSha256 = "sha256-HUgzoQuWBRnt8+lCiu9QfO1XR5EMnqVIkrL+nIMf0IA=";
```

new version, use `vendorHash`:
```suggestion
  vendorHash = "sha256-HUgzoQuWBRnt8+lCiu9QfO1XR5EMnqVIkrL+nIMf0IA=";
```